library/Faktory/Connection.hs

```diff
@@ -92,11 +94,12 @@ parseProvider =
   some (upperChar <|> char '_') <?> "an environment variable name"
 
 parseConnection :: Parser ConnectionInfo
-parseConnection = go <?> "tcp(+tls)://(:<password>@)<host>:<port>"
+parseConnection = go <?> "tcp(+tls)://(:<password>@)<host>:<port>(/namespace)"
  where
   go =
     ConnectionInfo
       <$> (False <$ string "tcp://" <|> True <$ string "tcp+tls://")
       <*> optional (char ':' *> manyTill anySingle (char '@'))
       <*> manyTill anySingle (char ':')
       <*> (read <$> some digitChar)
+      <*> optional (char '/' *> some anySingle)
```

Comment:
:ok_hand:

Reply:
Any day where I get to write a parser is a good day